error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/ImplicitBlockingFluxOperationTest.java

```diff
@@ -0,0 +1,140 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.FixChoosers;
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.jupiter.api.Test;
+
+final class ImplicitBlockingFluxOperationTest {
+  @Test
+  void identification() {
+    CompilationTestHelper.newInstance(ImplicitBlockingFluxOperation.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "import reactor.core.publisher.Flux;",
+            "",
+            "class A {",
+            "  void m() {",
+            "    // BUG: Diagnostic contains:",
```

Comment:
We can simplify this testing setup like this:
```suggestion
            "    // BUG: Diagnostic contains:",
              "class A {",
            "  void m() {",
            "    Flux.just(1).toIterable();",
            "    Flux.just(2).toStream();",
            "  }")
```

Additionally it would be nice to add some other test cases where we use `toIterable` and `toStream` on other objects then `Flux` to show it doesn't flag those.